xbmc/video/VideoDatabase.cpp

```diff
@@ -8029,6 +8048,9 @@ bool CVideoDatabase::GetMoviesByWhere(const std::string& strBaseDir, const Filte
     if (!videoUrl.FromString(strBaseDir) || !GetFilter(videoUrl, extFilter, sorting))
       return false;
 
+    if (!videoUrl.HasOption("videoversionid"))
+      extFilter.AppendWhere("isDefaultVersion = 1");
```

Comment:
I'm reviewing how the version folder works and I know it's a bit far already,.. Do you remember if there is a reason for this block not to be integrated in GetFilter() with other videoversionid option logic?
I'm thinking about moving it in upcoming PR.

Reply:
@CrystalP sorry, don't remember.